pkgs/by-name/po/polkit/package.nix

```
@@ -172,9 +175,13 @@ stdenv.mkDerivation rec {
         mv "$DESTDIR/''${!o}" "''${!o}"
     done
     mv "$DESTDIR/etc" "$out"
+    mv "$DESTDIR/usr/lib"/* "$out/lib"
```

With people merging stuff from r-ryantm without proper review of upstream changes, I am a bit paranoid. I think I would prefer more explicit 

```suggestion
    mkdir "$out/lib"
    mv "$DESTDIR/usr/lib/"{sysusers,tmpfiles}".d" "$out/lib"
```

to reduce the chance of new hardcoded artifact slipping through our fingers. The third one can be set by `-Dsystemdsystemunitdir`.

Or maybe we could convince upstream to add options for the `.d` as well.

---

How's that?

> Or maybe we could convince upstream to add options for the `.d` as well.

It was [hard enough](
https://github.com/polkit-org/polkit/pull/417#pullrequestreview-2057772070) to stop there being a hard dependency on systemd.pc.  Given that we need to have this moving mechanism for other stuff anyway, I don't think it's worth it.